Sources/HaCWebsiteLib/Controllers/LandingPageController.swift

```diff
@@ -29,9 +25,11 @@ struct LandingPageController {
     )
   ]
 
-  static let updates = eventPostCards + videos
-
   static var handler: RouterHandler = { request, response, next in
+    let eventPostCards : [PostCard] = EventServer.getCurrentEvents().flatMap{ event in 
```

Comment:
Hmm we are duplicating the post card getting behaviour here. We should perhaps have the LandingPageController instantiate a LandingPageUpdateFeedController and pass that down to the LandingPage view model. But that can happen in another PR if you wanna run ahead with this!